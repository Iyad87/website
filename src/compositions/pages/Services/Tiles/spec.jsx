import React from 'react';
import { shallow } from 'enzyme';
import ServicesTiles from './index';

test('compositions/pages/Services/Tiles snapshot', () => {
  const wrapper = shallow(<ServicesTiles />);

  expect(wrapper).toMatchSnapshot();
});
