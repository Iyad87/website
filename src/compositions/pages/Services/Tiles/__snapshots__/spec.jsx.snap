// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compositions/pages/Services/Tiles snapshot 1`] = `
<Container
  wide={true}
>
  <TileGrid>
    <styled__Tile>
      <styled__TileContent>
        <styled__TileContentTitle>
          About Us
        </styled__TileContentTitle>
        <styled__TileContentBody>
          We have team members from a broad spectrum of backgrounds and incredible technology companies.
        </styled__TileContentBody>
      </styled__TileContent>
    </styled__Tile>
    <styled__Tile
      colour="darker"
    >
      <styled__TileContent>
        <styled__TileContentTitle>
          Case Studies
        </styled__TileContentTitle>
        <styled__TileContentBody>
          XXX
        </styled__TileContentBody>
      </styled__TileContent>
    </styled__Tile>
  </TileGrid>
</Container>
`;
